testutil-server/src/main/java/io/spine/testing/server/blackbox/VerifyState.java

```diff
@@ -77,19 +81,49 @@ protected abstract void compare(ImmutableCollection<? extends Message> expected,
                                     ImmutableCollection<? extends Message> actual);
 
     /**
-     * Obtains a verifier which checks that the system contains exactly the passed entity states.
+     * Obtains the producer of {@link #exactly(TenantId, Class, Iterable)}.
```

Comment:
In this changeset you mix the verb `produce` with the term `producer` (the latter meaning an entity which is responsible for emitting an event or a command). I don't think it's fine.

In this particular piece of the documentation the whole text looks misleading. I.e. it's confusing for me as a user why I would want to use this method anyway. Besides, the `@return` section tells lies.